src/services/EventService.js

```diff
@@ -20,26 +20,25 @@ apiClient.interceptors.request.use(function (config){
 const userLogin = (payload, callback) => {
     apiClient.post('/auth/login', payload, callback)
         .then((res) => {
-            localStorage.setItem(TOKEN_KEY, res.data);
-            callback(true);
+            localStorage.setItem(TOKEN_KEY, res.data.token);
+            callback(true, res.data.name);
```

Comment:
The server should only be returning a `Bearer <token>` on a successful authentication. I don't think the `.token` and `.name` are accessible methods. Were you about to confirm the behaviour locally?

Reply:
Oh I see, (Backend-PR#6)[https://github.com/c9y7ip/ezpark-backend/pull/6] handles this